### PR TITLE
Feature/economy tests

### PR DESCRIPTION
**Pull request by lisamso**

### CODE REVIEW

File: economy/models.py [hunk 1]
```diff
@@ -42,7 +48,61 @@ def invalid(self):
...
+    def __str__(self):
+        return "A list of items purchased at date %s, registered at date %s by person %s" % \
+               (self.date_purchased, self.date_registered, self.signed_off_by.username)
+
+    def __repr__(self):
+        return "PurchaseList(date_purchased=%s, date_registered=%s, signed_off_by=%s, comment=%s)" % \
+               (self.date_purchased, self.date_registered, self.signed_off_by.username, self.comment)
+
+
+class Purchase(models.Model):
```
Comment (trmdy) [inline economy/models.py:91]: The only thing I feel missing in this model is a `get_purchase_sum` method which calculates the total sum of the purchase, i.e. `product.price * amount`.

Not essential, but could come in handy.

**Review by trmdy — APPROVED** [review]
Looks good!

We might think about squashing some of the migrations at some point, when we merge the economy PR into develop.

The comments left in the file can be safely ignored if you wish to.

**Comment by trmdy** [comment]
Note that the CI failure is due to an issue with Python 3.7. This same issue is why we have removed python3.7 from the supported platforms list. The PR succeeds for Python3.6.

**Comment by chdefrene** [comment]
Closing this since I have migrated the necessary changes into #28

**Pull request closed without merging**